README.md

```diff
@@ -9,7 +9,7 @@
 
 ## Needed Package
 
-- [black](https://github.com/psf/black) (22.6.0, optional)
+- [black](https://github.com/psf/black) (23.1.0, optional)
 - [flake8](https://github.com/PyCQA/flake8) (4.0.1, optional)
 - [isort](https://github.com/PyCQA/isort) (5.10.1, optional)
```

Comment:
Please update the version to `5.12.0`.

Reply:
Done in the commit: [af66f0b](https://github.com/lsst-ts/ts_m2com/pull/26/commits/af66f0bb11ebcfc7f66ecdff70f7470a6c826a94). Thanks!